purchase_order_line_sequence/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 Camptocamp SA - Damien Crier, Alexandre Fayolle
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Purchase Order Line Sequence",
+    "summary": "Adds sequence to PO lines and propagates it to"
+    "Invoice lines and Stock Moves",
+    "version": "13.0.1.0.0",
+    "category": "Purchase Management",
+    "author": "Camptocamp, "
+    "Eficent, "
```

Comment:
```suggestion
    "ForgeFlow, "
```
It should be changed everywhere:)

Reply:
Done!